src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.ts

```diff
@@ -37,13 +42,27 @@ interface EntityCountDashboardConfig {
     DashboardListWidgetComponent,
     EntityBlockComponent,
     NgIf,
+    MatButtonToggleGroup,
+    CommonModule,
+    FormsModule,
```

Comment:
I think we are now not using these in the final implemention, are we? Let's remove if not required.